### DISCUSSION TITLE
Using with eslint-config-prettier on save

### DISCUSSION THREAD

**Discussion by DmitriiKholodov** [trivago/prettier-plugin-sort-imports#85] [Q&A]
Hello there! I've tried to install the plugin, but I guess that the new options as 

`"importOrder": ["^@material-ui\\/(.*)$", "^@apollo\\/(.*)$"],
  "importOrderSeparation": true`

are ignored.

I don't have prettier 2.x.x in my dependencies I use eslint-config-prettier instead.

So can you please explain to me is it impossible to use it this way?

By the way, It is working if I install prettier globally and use it through CLI.

Thanks anyway!

**Comment by ayusharma** [comment]
HI @DmitriiKholodov, I just noticed your question here. I'll get back to you. FYI @byara

**Reply by DmitriiKholodov** [reply]
I’ve already solved this. By the way Thanks for your feedback!
